frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -124,31 +138,29 @@ function getParameterPanelBorderColor(theme?: DisplayTheme) {
 export const ParametersWidgetContainer = styled(FullWidthContainer)<{
   embedFrameTheme?: DisplayTheme;
   hasScroll: boolean;
+  canSticky: boolean;
   isSticky: boolean;
 }>`
   padding-top: ${space(1)};
   padding-bottom: ${space(1)};
 
   ${props =>
-    props.hasScroll &&
-    css`
-      border-bottom: 1px solid
-        ${getParameterPanelBorderColor(props.embedFrameTheme)};
-    `}
-
-  ${props =>
-    props.isSticky &&
+    props.canSticky &&
```

Comment:
Rename the old `isSticky` to `canSticky` and repurpose `isSticky` as a meaning for the filters is actually sticky right now.